test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
I'm not aware of any way to test it directly without using `cgi-fcgi`. I thought about the fact that the client image would be left behind, but I didn't make the fix. Should be easy enough to delete it along with the server container in the `trap`.

I can fix that later today. Let me know if you see any other changes you'd like made.